jenkins-scripts/dsl/ignition.dsl

```diff
@@ -225,11 +262,11 @@ Map supported_install_pkg_branches(String ign_software)
 {
   major_versions_prerelease = ignition_prerelease_pkgs["${ign_software}"]
 
-  // construct a map of stable packages based on supported_branches and
+  // construct a map of stable packages based on supported_ign_branches and
   // all_supported_distros
   map_of_stable_versions = [:]
   map_of_stable_versions[ign_software] = [:]
-  supported_branches(ign_software).each { major_version ->
+  supported_ign_branches(ign_software).each { major_version ->
```

Comment:
Do we need the equivalent for gz?

Reply:
this logic is used for creating the `-install` jobs for testing debian packages and homebrew bottles. We aren't quite ready for that, so I've added a `todo` comment to support `gz-*` branches after the garden release (https://github.com/gazebo-tooling/release-tools/pull/787/commits/f23a25051c795deb66610fe682f5d3483c10b927)